frontend/views/containers/chatroom/ChatMembersAllModal.vue

```diff
@@ -0,0 +1,434 @@
+<template lang='pug'>
+modal-base-template.has-background(ref='modal' :fullscreen='true' :a11yTitle='L("Channel members")')
+  .c-container
+    .c-header
+      div
+        i18n.is-title-2.c-title(
+          tag='h2'
+        ) Members
+        .c-description {{ attributes.name }} ∙ {{attributes.private ? L("Private channel") : L("Public channel")}}
```

Comment:
We should distinguish to the user the 3 different type of privacy levels.

So:

- For private channels: "Visibility: private channel"
- For group channels: "Visibility: group members only"
- For public channels: "Visibility: completely public!"

Please check the rest of the code to see if there's anywhere else where this also needs to be updated.

Reply:
Yeap. Let's distinguish in more detail.

- For private channels: `Private channel`
- For group channels: `Group members only`
- For public channels: `Public channel`

I don't think we should use `completely`. What do you think, @taoeffect?